defaults/main.yml

```diff
@@ -20,12 +20,12 @@ neofs_http__http_allow: []
 neofs_http__http_group_allow: []
 neofs_http__http_host_allow: []
 
-neofs_http__metrics_enabled: False
-#neofs_http__metrics_address: 'localhost'
-#neofs_http__metrics_port: 5662
-#neofs_http__metrics_allow: []
-#neofs_http__metrics_group_allow: []
-#neofs_http__metrics_host_allow: []
+neofs_http__prometheus_enabled: False
+#neofs_http__prometheus_address: 'localhost'
+#neofs_http__prometheus_port: 5662
+#neofs_http__prometheus_allow: []
+#neofs_http__prometheus_group_allow: []
+#neofs_http__prometheus_host_allow: []
```

Comment:
1. keep names in sync with other roles
2. comments should be removed, as in other roles

Reply:
Thanks, it has been fixed. Names in the other roles will be updated soon. 
Thanks.